crates/wysiwyg/src/dom/dom_list_methods.rs

```diff
@@ -94,6 +104,13 @@ where
         self.join_nodes_in_container(&handle.parent_handle());
     }
 
+    /// Extract items from the list at given handle and positions
+    /// and move them appropriately into the DOM. Extracted list
+    /// items are separated by line breaks.
+    ///
+    /// * `handle` - the list handle.
+    /// * `start` - child index at which the extraction should start.
```

Comment:
I personally find `start` a bit confusing, since I automatically associate it with the `start, end` parameters we usually use for selections. I'd change it to something like `from_child_index`, but that's probably subjective, it's ok to leave it as is.

Reply:
Yeah I wanted to change that too actually, I was thinking `start_index` before but it didn't felt like it was making things more clear. `from_child_index` makes some lines a bit long 😅 so I went with `child_index`, I think it's quite clear it's a position from which we are starting the operation (and if it's not, the documentation says so 🙂 ).